docs/en/writing-running-appium/ios-xctest-mobile-gestures.md

```diff
@@ -0,0 +1,182 @@
+## Automating Mobile Gestures For iOS with WDA/XCTest backend
+
+Unfortunately Apple's XCTest framework does not natively support W3C standards for
+TouchAction interface implementation. Although, it provides rich set of gestures,
+inluding these, that are unique for iOS platform. It is possible to directly invoke these
+gestures in Appium starting from version 1.6.4-beta.
+
+It is important to rememeber that XCTest and WDA are being constantly changed.
+This means all "mobile: *" commands can be also subject of change in Appium
+without any preliminary notice.
+
+
+### mobile: swipe
+
+This gesture performs a simple "swipe" gesture on the particular screen element or
+on the application element, which is usually the whole screen.
+
```

Comment:
can you please clarify how much swipe is done ? 
Is it full width or height of device ?
In case somebody wants to do partial swipe; in that case is this method useful ?